src/server/slowlog.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <boost/circular_buffer.hpp>
+#include <string>
+#include <vector>
+
+#include "base/integral_types.h"
+#include "server/common.h"
+namespace dfly {
+
+class SlowLogEntry {
+ public:
+  SlowLogEntry(int64_t command_id, int64_t unix_timestamp, int64_t execution_time_micro,
+               std::vector<std::string_view> cmd_args, std::string_view client_ip,
+               std::string_view client_name);
+
+  const std::vector<std::string_view>& CmdArgs() const {
+    return cmd_args_;
+  }
+  int64_t CommandId() const {
+    return command_id_;
+  }
+  int64_t UnixTimestamp() const {
+    return unix_timestamp_;
+  }
+  int64_t ExecutionTimeMicro() const {
+    return execution_time_micro_;
+  }
+  std::string_view const& ClientIp() const {
+    return client_ip_;
+  }
+  std::string_view const& ClientName() const {
+    return client_name_;
+  }
+
+ private:
+  std::vector<std::string_view> cmd_args_;
```

Comment:
sorry I missed this but this object must own all the strings.
`client_ip_` and `client_name_` here must be std::string.
you can not keep `cmd_args_` because these are references to 
strings that are going to disappear once a request finishes. 
If it's ok by you I will fix add this part to the PR.

Reply:
Okay by me!